src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Perhaps header comment can be added to describe what the interface represents.
Recall from tutorial that an interface is a behavioural contract, perhaps `Represents a printable entity ....`.